jenkins/workers.py

```diff
@@ -0,0 +1,94 @@
+"""
+Use the jenkins queue and computer APIs to log info about workers.
+
+Usage:
+    `python jenkins/workers.py -j https://example.jenkins.com`
+
+Example Output:
+    [INFO] Counts of workers and queue length from Jenkins API:
```

Comment:
I didn't realize/forgot this was a script that would be run in Jenkins. Let's just be sure it's allocated to a worker that will always have an executor available. Timestamps are not crucial so long as splunk gets them accurately. sounds like it might cause more noise than necessary so this is just an open question. Not a must.